src/main.c

```diff
@@ -7,6 +7,9 @@
 /* The inclusion must follow the fixed order, otherwise it fails to build. */
 #include "defs.h"
 
+/* Define target machine */
+#include "def.h"
```

Comment:
If the generated configurations are located in file `config.h`, then you should move it to the top of user headers.

Reply:
i.e.
```c
#include "config.h"

#include "globals.c"
#include "cfront.c"
```